Application/EdFi.Ods.Tests/EdFi.Ods.Common/Caching/XxHash3CodeTests.cs

```diff
@@ -0,0 +1,221 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using EdFi.Ods.Common.Caching;
+
+namespace EdFi.Ods.Tests.EdFi.Ods.Common.Caching;
+
+using NUnit.Framework;
+using Shouldly;
+using Standart.Hash.xxHash;
+
+[TestFixture]
+public class XxHash3CodeTests
```

Comment:
Optional: add test coverage for `null`, `ulong`, `long`, `byte[]`, `short`, and `Resource` types.